kapacitor/client.go

```diff
@@ -12,6 +12,10 @@ import (
 const (
 	// Prefix is prepended to the ID of all alerts
 	Prefix = "chronograf-v1-"
+
+	// DefaultFetchRate is the rate at which paginated responses will be consumed
```

Comment:
1. ack
2. I try not to put the name in because you have to refer to it as `kapacitor.FetchRate`. I've written too much Java to find `kapaRate := kapacitor.KapacitorFetchRate` appealing :) I'm cool with the comment though.